src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="img"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo__Img"
```

Comment:
The `alt` attribute should describe the image content. Instead of 'Moyo__Img', it should be a description of what the logo represents or the name of the company. For example, 'Moyo Logo'.